src/OrchardCore.Modules/OrchardCore.ContentFields/Drivers/LinkFieldDisplayDriver.cs

```diff
@@ -81,8 +84,16 @@ public override async Task<IDisplayResult> UpdateAsync(LinkField field, IUpdateM
                     updater.ModelState.AddModelError(Prefix, S["The text default value is required for {0}.", context.PartFieldDefinition.DisplayName()]);
                 }
 
-                // Run this through a sanitizer in case someone puts html in it.
-                // No settings.
+                if (!string.IsNullOrEmpty(field.Url) || !string.IsNullOrEmpty(field.Text))
+                {
+                    // Test the input against the sanitizer.
+                    var aTag = @"<a href=""" + (field.Url ?? "") + @""">" + (field.Text ?? "") + "</a>";
```

Comment:
Use `string.Empty` if it's possible

Reply:
In this case it would make the code less readable. And the compiler handles that for you.